/robot_arm:
  ros__parameters:
    serial_port: "/dev/ttyACM0"
    baud_rate: 115200
    sample_interval_s: 0.2
    
    # You can add more parameters here

